index.js

```diff
@@ -71,9 +71,14 @@ var serialNumber = function (cb, cmdPrefix) {
 		cmd = 'system_profiler SPHardwareDataType | grep ';
 		break;
 
-	case 'linux':
 	case 'freebsd':
-		cmd = 'dmidecode -t system | grep ';
+	case 'linux':
```

Comment:
There is no separate freebsd section. This is the syntax for giving two cases the same logic block.